dev/com.ibm.websphere.appserver.features/visibility/public/messagingServer-2.0/io.openliberty.messagingServer-2.0.feature

```diff
@@ -0,0 +1,21 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.messagingServer-2.0
```

Comment:
As mentioned elsewhere, please can we make this 3.0 instead of 2.0?

Reply:
Yes, we've the green light on bumping all jakarta messaging features to version 3.0.